notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -11594,6 +11594,45 @@ opposition to authority [Neo's boss at the software company inside the Matrix co
+https://en.wikipedia.org/wiki/Existenz
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+speculative virtual reality [A main novelty of the film is the titular virtual reality game eXistenZ. People connect to the game via "bio-ports" in their spines, and once plugged in it is so realistic that the lines between where the game leaves off and reality begins hopelessly blurred.]
+
+:: Major Themes
+what if I couldn't trust the veracity of my senses [Allegra and Ted quickly ran into trouble distinguishing what was real from the goings on in the virtual reality game in which they were immersed.]
+what if I was trapped inside a game [Allegra and Ted were plugging in and out from multiple nested virtual reality worlds within worlds, never quite knowing in the end whether they'd made it back to the real world.]
+speculative biotechnology [The film is set in a near-future where biotechnological virtual reality game consoles known as "game pods" have replaced electronic ones. The game pods are grotesque biological organisms of some sort with umbilical cord-like connectors that plug into special surgically implanted "bio-ports" in the players' spines.]
+brain-computer interface [Ted reluctantly underwent a back alley surgical procedure to have a "bio-port" installed in his lower spine. This allowed him to connect to a highly realistic virtual reality game. In general, all of the virtual reality game enthusiasts in the film had "bio-ports" installed in their spines.] 
+celebrity worship [A gas station operator dropped down on his hands and knees before the world renowned game designer Allegra Geller upon realizing it was really her.]
+terrorism [A group of fanatics called the Realists were fighting two major virtual reality gaming companies to prevent them from "deforming" of reality.]
+political assassination [The world renowned VR game designer Allegra Geller was being targeted for assassination by a group of fanatics who were trying to prevent the "distorting" of reality.]
+
+:: Minor Themes
+video gaming [The world renowned game designer Allegra Geller was shown testing her new virtual reality game, eXistenZ, to a focus group. Allegra encouraged an ambivalent Ted to plug into her latest virtual reality game.]
+speculative weapon [A would-be assassin shot Allegra in the shoulder with a tooth shooting organic pistol that he'd managed to sneak though a security checkpoint.]
+unethical business practices [The Cortical Systematics game company tried to steal the titular virtual reality game eXistenZ from their rival company Antenna Research, and also at one point used a double agent to try to get Allegra to defect from Antenna Research to their company.]
+ethics [Ted confronted Allegra about her needlessly shooting dead a virtual reality Chinese waiter who was pleading for his life, asking of her if she were going to kill him (Ted) next. In so doing, the viewer is made to ponder about whether it is okay to behave in a sadistic manner even in a simulated setting.]
```

Comment:
Sounds like https://www.themeontology.org/theme.php?name=killing%20an%20artificial%20person  but at any rate I think we have similar stories - maybe under AI rights, and also that one with a guy reliving a day over again thousands of times

Reply:
Good call. I happily replaced **ethics** with **killing an artificial person**.

Funny you mention about the story with the guy who was reliving the same day over and over again. I grappled with using the **the ethics of contemplating an immoral act** theme we coined for that story, but the situation in eXistenZ didn't quite fit it in my ultimate estimation.